cmd/submit_test.go

```diff
@@ -534,6 +537,46 @@ func TestSubmitRelativePath(t *testing.T) {
 	assert.Equal(t, "This is a file.", submittedFiles["file.txt"])
 }
 
+func TestSubmitServerErr(t *testing.T) {
+	handler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusBadRequest)
+		fmt.Fprintf(w, `{"error": {"type": "error", "message": "test error"}}`)
+	})
+
+	errts := httptest.NewServer(handler)
+	defer errts.Close()
+
+	tmpDir, err := ioutil.TempDir("", "secret-temp-dir")
```

Comment:
Let's give this a prefix that reflects the purpose of the test. Maybe something like `submit-error`